tests/test_clients.py

```diff
@@ -45,3 +45,42 @@ def test_close_without_clients(hlwm):
 def test_close(hlwm, running_clients_num):
     hlwm.create_clients(running_clients_num)
     hlwm.call('close')
+
+
+@pytest.mark.filterwarnings("ignore:tostring")
+@pytest.mark.parametrize("urgent", [True, False])
+def test_urgent_on_start(hlwm, x11, urgent):
+    # first create a dummy window, such that the second
+    # window is not focused and thus keeps the urgent flag
+    winid_focus, _ = hlwm.create_client()
+    window, winid = x11.create_client(urgent=urgent)
+    assert x11.is_window_urgent(window) == urgent
+    assert hlwm.get_attr('clients.{}.urgent'.format(winid)) \
+        == hlwm.bool(urgent)
+
+
+@pytest.mark.filterwarnings("ignore:tostring")
+def test_urgent_after_start(hlwm, x11):
+    winid_focus, _ = hlwm.create_client()
+    winid, _ = hlwm.create_client()
+    assert hlwm.get_attr('clients.{}.urgent'.format(winid)) == 'false'
+    assert not x11.is_window_urgent(x11.window(winid))
+
+    # make the client urgent:
+    x11.make_window_urgent(x11.window(winid))
+
+    assert hlwm.get_attr('clients.{}.urgent'.format(winid)) == 'true'
+    assert x11.is_window_urgent(x11.window(winid))
+
+
+@pytest.mark.parametrize("explicit_winid", [True, False])
+def test_urgent_jumpto(hlwm, x11, explicit_winid):
```

Comment:
Test name suggestions: `test_urgent_jumpto_makes_client_nonurgent` or `test_urgent_jumpto_resets_urgent_flag`

Reply:
I've renamed it!